packages/web-components/package.json

```diff
@@ -19,14 +19,11 @@
   "scripts": {
     "build": "stencil build --docs-json component-docs.json",
     "build-bindings": "tsc -p tsconfig.bindings.json",
-    "start": "concurrently -n storybook,stencil -c \"bgMagenta.bold,bgBlue.bold\" \"npm:storybook\" \"npm:watch:stencil\"",
```

Comment:
These were leftover dependencies when we had a separate development storybook for the `web-components` branch.